### PR TITLE
COMMERCE-8925 removed aui:script from jsp

### PR DESCRIPTION
**Pull request by garofalog**

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/price_range_facets/view.jsp [hunk 1]
```diff
@@ -132,49 +132,20 @@ CPPriceRangeFacetsDisplayContext cpPriceRangeFacetsDisplayContext = (CPPriceRang
 						<aui:input cssClass="price-range-input" label="<%= StringPool.BLANK %>" name="maximum" prefix="<%= HtmlUtil.escape(cpPriceRangeFacetsDisplayContext.getCurrentCommerceCurrencySymbol()) %>" type="number" value="<%= max %>" wrapperCssClass="col-md-5 price-range-input-wrapper" />
 
 						<div class="col-md-3 ml-2 p-0">
-							<button class="btn btn-secondary price-range-btn" onclick="<%= liferayPortletResponse.getNamespace() + "submitPriceRange();" %>"><liferay-ui:message key="go" /></button>
+							<button class="btn btn-secondary price-range-btn" id="submitPriceRange"><liferay-ui:message key="go" /></button>
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/price_range_facets/view.jsp:135]: Rename it to "priceRangeButton" and prefix it with namespace?

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@garofalog Take a look at the comments, thanks!

**Comment by garofalog** [comment]
Thank you!

**Comment by garofalog** [comment]
Thank you!

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+		searchParams.set('basePrice', '[ ' + min + ' TO ' + max + ' ]');
+		searchParams.set('max', max);
+		searchParams.set('min', min);
+
+		url.search = searchParams;
+
+		navigate(url.toString());
+	};
+	const submitPriceRange = document.getElementById('priceRangeButton');
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js:51]: Rename variable name to "priceRangeButton".

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/price_range_facets/view.jsp [hunk 3]
```diff
@@ -132,49 +132,20 @@ CPPriceRangeFacetsDisplayContext cpPriceRangeFacetsDisplayContext = (CPPriceRang
 						<aui:input cssClass="price-range-input" label="<%= StringPool.BLANK %>" name="maximum" prefix="<%= HtmlUtil.escape(cpPriceRangeFacetsDisplayContext.getCurrentCommerceCurrencySymbol()) %>" type="number" value="<%= max %>" wrapperCssClass="col-md-5 price-range-input-wrapper" />
 
 						<div class="col-md-3 ml-2 p-0">
-							<button class="btn btn-secondary price-range-btn" onclick="<%= liferayPortletResponse.getNamespace() + "submitPriceRange();" %>"><liferay-ui:message key="go" /></button>
+							<button class="btn btn-secondary price-range-btn" id="priceRangeButton"><liferay-ui:message key="go" /></button>
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/price_range_facets/view.jsp:135]: Prefix ID with the namespace.

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@garofalog Please take a look at the comments, thanks!

**Comment by garofalog** [comment]
All changes requested were done @alessiorendina

**Comment by garofalog** [comment]
All changes requested were done @alessiorendina

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+		searchParams.set('basePrice', '[ ' + min + ' TO ' + max + ' ]');
+		searchParams.set('max', max);
+		searchParams.set('min', min);
+
+		url.search = searchParams;
+
+		navigate(url.toString());
+	};
+	const submitPriceRange = document.getElementById(
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js:51]: Rename variable to "priceRangeButton".

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@garofalog Please take a look at the comment, thanks!

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js (hunk 4 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/price_range_facets/view.js:51]: @garofalog Please note that this one is not resolved, thanks.

**Review by alessiorendina — CHANGES_REQUESTED** [review]

**Comment by garofalog** [comment]
i could not get which change was requested. i think was `[${min}TO${max}]` and just pushed it. @alessiorendina  let me know of more changes are required. ty

**Comment by garofalog** [comment]
i could not get which change was requested. i think was `[${min}TO${max}]` and just pushed it. @alessiorendina  let me know of more changes are required. ty

**Comment by alessiorendina** [comment]
> i could not get which change was requested. i think was `[${min}TO${max}]` and just pushed it. @alessiorendina let me know of more changes are required. ty

@garofalog No, this is not what I requested. Please revert this change. What I requested is a variable name change from "submitPriceRange" to "priceRangeButton".

**Comment by garofalog** [comment]
Sorry, but your code's selection in comments here were not clear. Anyway i renamed the id yet, and i also renamed the variable's name right now.
About basePrice value is fine as it is, concatenated with backticks.

**Comment by garofalog** [comment]
Sorry, but your code's selection in comments here were not clear. Anyway i renamed the id yet, and i also renamed the variable's name right now.
About basePrice value is fine as it is, concatenated with backticks.

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:forward

**Pull request closed without merging**